jcl/src/openj9.dtfj/share/classes/com/ibm/jvm/j9/dump/extract/Main.java

```diff
@@ -367,20 +353,18 @@ private static void ensure(boolean condition, String errorMessage) {
 	}
 
 	private static void usageMessage(String message, int code) {
-		report("Usage: jextract dump_name [output_filename] [options]"); //$NON-NLS-1$
-		report(" output filename defaults to dump_name.zip"); //$NON-NLS-1$
-		report(" options:"); //$NON-NLS-1$
-		report("   -f executable_name override executable name"); //$NON-NLS-1$
-		report("   -help              print this usage message"); //$NON-NLS-1$
-		report("   -v                 enable verbose output"); //$NON-NLS-1$
-		if (_throwExceptions) {
-			throw new JExtractFatalException(message, code);
-		} else {
-			if (message != null) {
-				report(message);
-			}
-			System.exit(code);
-		}
+		report("Usage: jextract [options] dump_name [output_filename]"); //$NON-NLS-1$
+		report("  output filename defaults to dump_name.zip"); //$NON-NLS-1$
+		report("  options:"); //$NON-NLS-1$
+		report("    -e                  throw exceptions instead of calling System.exit()"); //$NON-NLS-1$
+		report("    -f executable_name  override executable name"); //$NON-NLS-1$
+		report("    -help               print this usage message"); //$NON-NLS-1$
+		report("    -interactive        start an interactive session"); //$NON-NLS-1$
+		report("    -p prefix           prefix for all paths (absolute or relative)"); //$NON-NLS-1$
+		report("    -r                  relax version checking"); //$NON-NLS-1$
+		report("    -v                  enable verbose output"); //$NON-NLS-1$
+		report("    --                  mark end of options"); //$NON-NLS-1$
```

Comment:
I don't understand the purpose of this, perhaps a more detailed message is appropriate.

Reply:
This is a commonly supported option with the meaning that subsequent arguments are not treated as options (even if they begin with `-`). A (perhaps unlikely) example:
```
jextract -r -- -v
```
considers the file named `-v` to be the core file.

Reply:
I think it's a GNU convention; see the [bash manual](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Invoking-Bash), for example.